routes/events.js

```diff
@@ -18,6 +18,8 @@ router.route('/:eventId/guests')
 router.route('/:eventId/guests/:guestId')
   .get(verifyToken, guestController.getGuestInfo)
   .put(verifyToken, guestController.updateGuest);
-// TODO: implement mailgun service to send approved email or check qr here
+
+router.route('/:eventId/guests/qr')
+  .put(verifyToken, guestController.verifyQR);
```

Comment:
QR is a thing from Front-end. API just know about the `ticketCode`, then the name `/:eventId/guests/qr` or `guestController.verifyQR` is not the best name. Use `/:eventId/guests/:guestId/tickets`

Reply:
For now, QR code will encode an object:
```
{
  guestId: 
  ticketCode:
}
```
And front-end will call:
`PUT /events/:eventId/guests/:guestId/tickets`
Body: `{ action: "something", ticketCode: 123456 }`

Reply:
Plz check this: https://github.com/EasyEventV2/ee2_api/pull/9/commits/49e714466a2bc0290fcb862712ff82060c14727b